include/eve/module/core/regular/frac.hpp

```diff
@@ -18,7 +18,7 @@ namespace eve
 {
 
   template<typename Options>
-  struct frac_t : elementwise_callable<frac_t, Options>
+  struct frac_t : elementwise_callable<frac_t, Options, raw_option>
```

Comment:
why is this in this PR ?

Reply:
because the the use of [] make useful yhe introduction of a raw version as it has consequences on the conservation of the zero sign in corner cases